doc/source/projections.rst

```diff
@@ -22,7 +22,7 @@ The same CRS can often be referred to in many ways. For example, one of the most
 commonly used CRS is the WGS84 latitude-longitude projection. This can be
 referred to using the authority code ``"EPSG:4326"``.
 
-*geopandas* can accept anything accepted by `pyproj.CRS.from_user_input() <https://pyproj4.github.io/pyproj/stable/api/crs.html#pyproj.crs.CRS.from_user_input>`_:
+*geopandas* can accept anything accepted by :meth:`pyproj.crs.CRS.from_user_input`:
```

Comment:
```suggestion
*geopandas* can accept anything accepted by :meth:`pyproj.CRS.from_user_input() <pyproj.crs.CRS.from_user_input>`:
```

I prefer that the geopandas users see the name as they should use it (so pyproj.CRS, not pyproj.crs.CRS)

Reply:
It needs to be that way to work with interspinx. The other option would be to use a link like what was done before.

Reply:
Note that I didn't change the actual link (that still uses pyproj.crs.CRS), only how it is displayed (rst syntax : ``` :meth:`displayed <link>`  ```)

Reply:
Just seeing this - I wasn't aware of that. Will reopen.

Reply:
Looks like you beat me to it :)

Reply:
I completely missed that you did that. :man_facepalming:. I changed the others to match and changed that back.

Reply:
Oh, suggested change. Wasn't there before.